website/www/site/data/io_matrix.yaml

```diff
@@ -219,6 +219,12 @@ categories:
           - language: java
             name: org.apache.beam.sdk.io.aws2.sns.SnsIO
             url: https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/io/aws2/sns/SnsIO.html
+      - transform: Pub/Sub Lite
+        description: I/O transforms for reading from Google Pub/Sub Lite.
+        implementations:
+          - language: java
+            name: org.apache.beam.sdk.io.pubsublite.PubSubLiteIO
+            url: https://beam.apache.org/releases/javadoc/2.36.0/org/apache/beam/sdk/io/gcp/pubsublite/PubsubLiteIO.html
```

Comment:
Should this be using the "current" slug instead of "2.36.0"?

Reply:
Changed.